src/tests/suite/daos_rebuild.c

```diff
@@ -1899,38 +1919,48 @@ static const struct CMUnitTest rebuild_tests[] = {
 	{"REBUILD23: disconnect pool during scan",
 	 rebuild_tgt_pool_disconnect_in_scan, NULL, test_case_teardown},
 	{"REBUILD24: disconnect pool during rebuild",
-	 rebuild_tgt_pool_disconnect_in_rebuild, NULL, test_teardown},
+	 rebuild_tgt_pool_disconnect_in_rebuild, NULL, test_case_teardown},
```

Comment:
oh, this seems to be changed to rebuild_sub_teardown,  i.e. it needs to cleanup the pool.

Reply:
Why ? My understanding is that the current/main/group opened pool must not be cleaned between sub-tests, only the ones that are used per sub-test and created by rebuild_sub_stup() if used.